src/Button.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import styled from "styled-components";
+
+function Button(props) {
+  return (
+    <StyledButton type={props.type} value={props.value} {...props}>
```

Comment:
If you are passing props to styled components avoid passing it through prop spreading... this works but much clearer to read and understand. 

```
<StyledButton 
   cssOptions={props.cssOptions} 
   type={props.type} 
   value={props.value} 
   {...props}
>
```

Reply:
Also if you are prop spreading `type` and `value` are default props... you could leave them in the prop spread.

```
<StyledButton 
   cssOptions={props.cssOptions} 
   {...props}
>
```